community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
+
+5. Humans are not rational, so don't expect them to behave rationally. We are totally full of feelings.
+
+6. We don't accept snowflake argument. Some people have thick skin and are borderline stoic, and they think that "I am responsible for my feelings" and on the opposite end of emotional spectrum, are people who think that "You don't get to decide if you hurt someone". We will try to have an inclusive space for both of these people.
+
+7. Humans hate to forgive people if forgiving is tough.
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
+
+## Basic Protocol in case you are harassed/bullied/found something offensive/weird/unacceptable
+
+Subject -> Who has been harassed/bullied
+
+Type 1 :- Constitutionally Legal Offense
+
+1. In this case the team is requested not to micromanage. let it go through official channels of college and judiciary and let them deal with it.
+
+Type 2 :- Everything Else
+
+1. If it has happened for the first time, dm the person. And let {pronoun} know that this behaviour is offensive to the subject. If not comfortable confronting then **????????????? @team comments**
+2. Post on a public forum that the subject finds this particular behaviour offensive/xyz. Name calling doesn't help.
+3. If it's repetitive, **what to do ???????????????????? @team , comments**
+
+## Basic Protocol in case you are accused of harassment/bullying/offending/doing weird shits/unacceptable behaviour
+
+Subject -> Who has been accused
+
+1. Well the subject can start by saying sorry, and ensuring that similar behaviour is not repeated in front of the person who has accused the subject "ever". If it's ingrained in the subject’s habit let the accussee know that, it's in your habit you will make sincere effort to decrease the frequency. The subject can take a note and share updates of progress {pronoun} have made on that person's feedback.
+2. Adding "genuinely" prefix in front of sorry doesn't help.
+3. It doesn't matter if the subject’s sorry or change is genuine or fake, as long as it's change it's good.
+
+## Basic Protocol in case you have noticed harassment.
+
+**@team comments**
+
+
+
+## Basic Protocol in case you are an executive/advisor and have to take action
+
+**@team comments**
```

Comment:
If it's comes under legal offense, advise the victim to directly deal with the authorities and provide all the necessary support. Time is crucial in such cases and the sooner the matter is pursued for prosecution the better.

If it doesn't come under legal offense, Inform other team members, see about their perspective if they have also witnessed the incident or any other incidents with same accused. Make sure the discussion doesn't continue indefinitely.

Mediate the conversation between the victim and the accused. The accused may try gaslighting as a way out.

If :
1. It is a single incident and the accused is ready for amends
2. The victim is fine with the amends and doesn't want to take this further

then the matter is over. However, if the victim is not fine with the amends, ask them to suggest what will be fine. If a settlement is not reached, suggest the victim to make the accusation public, if it helps.

With majority approval of advisors and substantiated proof of the incident is enough to terminate the accused from KOSS. The reason of termination should be made public to stop rumors and misinformation.

Reply:
Totally agreed, with a small suggestion.

> suggest the victim to make the accusation public

Also, help the victim to make the accusation public with or without redacting the names.